packages/xdl/src/Webpack.js

```diff
@@ -18,8 +18,8 @@ import ip from './ip';
 
 import type { User as ExpUser } from './User'; //eslint-disable-line
 
-const HOST = '0.0.0.0';
-const DEFAULT_PORT = 19006;
+const HOST = process.env.IP || '0.0.0.0';
```

Comment:
If you add this functionality you'll need to account for the URL getter https://github.com/expo/expo-cli/blob/f5d61b6fcc87cadaefc578ee770185fcc69fabff/packages/xdl/src/Webpack.js#L117-L125. This may need to read from the running instance of Webpack to ensure the correct IP is being used. Currently it's hardcoded.